data/_ui/tooltips.txt

```diff
@@ -293,6 +293,42 @@ tip "repair delay:"
 
 tip "disabled repair delay:"
 	`The number of seconds it takes for hull repairs to begin after being disabled.`
+	
+tip "disabled recovery timer:"
```

Comment:
You have "disabled recovery **time**" in other places, so either correct it here or change it in the code. Although I'd suggest renaming to "disabled recovery delay" to match repair and shield gen delays.